content/en/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md

```diff
@@ -346,7 +346,7 @@ export KUBECONFIG=$KUBECONFIG:$HOME/.kube/config
 ```
 ### Windows Powershell
 ```shell
```

Comment:
Does it also make sense to mark this as powershell rather than (Bourne) shell?

Reply:
It can't hurt.  I'm not sure if the docs site will reflect the differences in syntax highlighting but Github will.